vendor.mod

```diff
@@ -24,7 +25,7 @@ require (
 	github.com/aws/smithy-go v1.13.5
 	github.com/cloudflare/cfssl v1.6.4
 	github.com/containerd/cgroups/v3 v3.0.2
-	github.com/containerd/containerd v1.7.11
+	github.com/containerd/containerd v1.7.12-0.20231228191706-ce21450bb444 // releease/1.7 branch (v1.7.12-dev), includes https://github.com/containerd/containerd/pull/9569
```

Comment:
Updated to tip of release/1.7 (v1.17.12-dev) which includes https://github.com/containerd/containerd/pull/9569